src/Toast/story.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Toast from "./index";
+
+storiesOf("Toast", module)
+  .addDecorator((getStory) => (
+    <div className={"flex h-full items-center justify-center w-full mt-20"}>
+      {getStory()}
```

Comment:
I think we should add an example of how the toast can be triggered from a button, for example.

Reply:
done